base/Command.js

```diff
@@ -53,10 +53,14 @@ class Command {
   }
 
   async verifyUser(message, user, options = {}) {
-    const match = /(?:<@!?)?([0-9]{17,20})>?/gi.exec(user);
-    if (!match) throw new ParseError("Invalid Mention or ID", options.msg);
-    const id = match[1];
-    return this.client.users.fetch(id);
+    let match;
+    const idMatch = /(?:<@!?)?([0-9]{17,20})>?/gi.exec(user);
+    const nameMatch = /./gi.exec(user);
+    if (!idMatch && !nameMatch) throw new ParseError("Invalid Mention or ID", options.msg);
+    user.startsWith("<@") && user.endsWith(">") ? match = user.substr(2, 18) : match = user;
```

Comment:
This isn't needed? as the regex on line 57 will detect the user's ID from the mention anyway as you can see here ![](https://i.imgur.com/AzIFQbC.png), just grab (iirc) `match[1]` and you're golden.

Reply:
Oh, if thats the case that simplifies about 2 lines..